SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -603,6 +603,7 @@ void USpatialNetDriver::OnActorSpawned(AActor* Actor)
 	if (!Actor->GetIsReplicated() ||
 		Actor->GetLocalRole() != ROLE_Authority ||
 		!Actor->GetClass()->HasAnySpatialClassFlags(SPATIALCLASS_SpatialType) ||
+		!IsReady() ||
```

Comment:
If we're not ready - when will this code be re-run for this actor?

Reply:
It should be run on all actors in the level as part of CleanupLevelInitializedNetworkActors.

Reply:
Maybe I'm missing something, but I think this only gets called when actors are dynamically spawned, and is used to delete actors who shouldn't have been spawned on this server. I don't think this logic is caught in `USpatialGameInstance::CleanupLevelInitializedNetworkActors`. cc: @mattyoung-improbable

Reply:
This should only happen for dynamically spawned actors. Is there a chance that some logic will Spawn an actor _before_ IsReady is true?

Reply:
I don't really know all the ways that actors can be spawned. I know that NWX was hitting this even with the other race condition fixed. I can follow up with them and see what the actor hitting it was. 

I was actually hoping that once we had Offloading following the load balancing code path, we'd be able to remove this entirely.

Reply:
Mike and I discussed offline and I've added in a Warning log if a user creates an actor before the SpatialNetDriver is ready. This can be removed when we are able to force level load only after the connection.